fe1-web/src/features/home/objects/ConnectToLao.ts

```diff
@@ -37,6 +37,16 @@ export class ConnectToLao {
 
     return new ConnectToLao({
       ...obj,
+      /* in the future the qr code should encode a list of servers */
+      servers: [obj.server],
```

Comment:
Hmmm..  Apparently QR codes can hold up to 3 KB of information. I guess we would need to compute the size in bytes to throw an accurate warning as it will also depend on the length of the server addresses.. I think this should be doable using Buffers 🤔

Reply:
Hmm okay apparently it is not so easy because it also depends on the error correction level and the version used 😬